content/release-10-18-0/platform-10-18-0.md

```diff
@@ -243,6 +243,30 @@ Build version</th>
 </tr>
 </thead><tbody>
 
+<tr>
+<td>Administration</td>
+<td>Fixed an issue when cloning some of the default global roles (for example, "devices").</td>
+<td>MTM-45858</td>
+<td>10.18.0.98</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Administration</td>
+<td>Fixed an issue where the <b>Clear</b> button on the <b>Usage statistics</b> page failed to remove applied filters.</td>
+<td>MTM-50302</td>
+<td>10.18.0.93</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Administration</td>
+<td>The button name for confirming the revocation of all tokens has been revised to <b>Log out all users and invalidate tokens</b>.</td>

```

Comment:
what was it before? is it relevant?

Reply:
You mean the entire entry? I was also wondering if it is relevant.

Reply:
i mean, is it relevant (for the release notes entry) what it was before?

Reply:
No, not really, it was quite similar (I think it was "Invalidate tokens") so the context should be clear.